packages/tupaia-web/src/api/mutations/useExportDashboard.tsx

```diff
@@ -24,7 +24,7 @@ export const useExportDashboard = ({ onSuccess }: { onSuccess?: (data: Blob) =>
         selectedDashboardItems: selectedDashboardItems?.join(','),
       });
       // Auth cookies are saved against this domain. Pass this to server, so that when it pretends to be us, it can do the same.
-      const cookieDomain = new URL(import.meta.env.REACT_APP_TUPAIA_WEB_API_URL).hostname;
```

Comment:
Got caught out by not having this env var set during testing. So referenced the value in `api.ts`